addon/templates/components/bsy-checkbox.hbs

```diff
@@ -1,7 +1,8 @@
 <label>
-  {{one-way-checkbox
-    isChecked=isChecked
-    update=update
+  {{input
+    type="checkbox"
+    checked=isChecked
+    onclick=update
```

Comment:
`(action update)` ?
and why update? if this is an event name it `onUpdate` and assign a default function to it